src/arns.lua

```diff
@@ -353,6 +359,9 @@ function arns.getRegistrationFees()
 	return fees
 end
 
+--- Gets the token cost for an intended action
+--- @param intendedAction table The intended action
```

Comment:
Might be worth somehow specifying what the expected table keys are. I know the tooling likely won't pick it up, but it might still help the reader.